pages/docs/concepts/asyncapi-document/_section.md

```diff
@@ -0,0 +1,4 @@
+---
+title: AsyncAPI-document
```

Comment:
```suggestion
title: AsyncAPI document
```

Why did you put a dash there? Notice how weird it looks live, too 😂

<img width="243" alt="Screen Shot 2023-08-31 at 5 56 03 PM" src="https://github.com/asyncapi/website/assets/19964402/f1fbe0fa-d62a-450a-bd76-ea213f57601a">